docker-compose.local.yaml

```diff
@@ -22,7 +22,6 @@ services:
     extra_hosts:
       - 'host.docker.internal:host-gateway'
   loki:
-#    image: grafana/loki:main-12cd12a
```

Comment:
FWIW. not sure if intended.

Reply:
Yeah it was intended, although it could be moved to another PR